### PR TITLE
Allow changing Traveler Boots speed using ThaumicBoots Keybinds.

### PR DESCRIPTION
**Pull request by Drathonix**
# Changes
EMT Traveler boots now implement IBoots (a new API feature I added in a PR on the ThaumicBoots repo)

EMT Traveler boots can now have their jump height and speeds changed.

[Thaumic Boots Pull Request](https://github.com/GTNewHorizons/ThaumicBoots/pull/27)

### CODE REVIEW

**Comment by Drathonix** [comment]
I've been having issues with maven local repos (literally no clue why I'm doing exactly what should be done) I've temporarily included the custom ThaumicBoots jar in the submission.

**Comment by Dream-Master** [comment]
@Drathonix not forget to remove it after testing. i can do  pre tag on thaumic boots repo if you want?

File: lib/ThaumicBoots-1.2.1-master+4e33b9941e-dirty-dev.jar
Comment (NotAPenguin0) [inline lib/ThaumicBoots-1.2.1-master+4e33b9941e-dirty-dev.jar:1]: Why is this file included?

**Review by NotAPenguin0 — CHANGES_REQUESTED** [review]

File: lib/ThaumicBoots-1.2.1-master+4e33b9941e-dirty-dev.jar
Comment (NotAPenguin0) [reply lib/ThaumicBoots-1.2.1-master+4e33b9941e-dirty-dev.jar:1]: Didn't read above, sorry.

**Comment by Drathonix** [comment]
@Dream-Master A pre tag would work.

**Comment by Alastors** [comment]
"A new API feature" you just changed the name for no reason

**Review by Alastors — DISMISSED** [review]
Mitch has already said no to Thaumic Boots being a hard dep

**Comment by Dream-Master** [comment]
@Drathonix https://github.com/GTNewHorizons/ThaumicBoots/releases/tag/1.2.2-pre

**Review by Alastors — CHANGES_REQUESTED** [review]
Accept the PR to your branch and I'll approve and merge

**Review by Alastors — APPROVED** [review]

**Pull request merged**